scripts/platforms/greenhouse_apply_v2.py

```
@@ -0,0 +1,550 @@
+        except Exception:
+            label_text = ""
+
+    if not label_text:
+        try:
+            label_text = normalize_space(
+                await field.evaluate(
+                    """
+                    (el) => {
+                      const labelledBy = el.getAttribute('aria-labelledby');
+                      if (labelledBy) {
+                        const ids = labelledBy.split(/\s+/).filter(Boolean);
```

Ruff (0.15.12)

[warning] Invalid escape sequence: `\s`

Use a raw string literal

(W605)